src/common/ContractInteractor.ts

```diff
@@ -130,15 +131,18 @@ export default class ContractInteractor {
   getProvider (): provider { return this.provider }
 
   async init (): Promise<void> {
+    log.debug('Contract Interactor - Initializing')
     if (this.isInitialized()) {
       throw new Error('_init was already called')
     }
     await this._initializeContracts()
+    log.debug('Contract Interactor - Already initialized')
```

Comment:
```suggestion
    log.debug('Contract Interactor - Initialized successfully')
```
suggesting this because saying "already" might be interpreted as what's happening on line `136`

not a blocker